.github/workflows/dev-build.yml

```diff
@@ -76,18 +76,16 @@ jobs:
         with:
           node-version: "12"
 
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-      - uses: actions/cache@v2.1.3
-        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+      - name: Cache node_modules
+        uses: actions/cache@v2.1.3
+        id: cached-node_modules
         with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
+          path: |
+            node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
```

Comment:
Might want to keep "yarn" in the key if you decide you want to cache the Poetry stuff later. Unlikely they'll collide, but could if it's just the hash

Reply:
But the `yarn.lock` hash will make sure that that's never colliding.

Reply:
Potentially the could collide as "OS_NAME-SHA_OF_YARNLOCKS" could match "OS_NAME-SHA_OF_POETRYLOCKS"
Its pretty unlikely, but the SHAs of the lock files could be the same as it's just an (MD5?) hash of the file(s) https://www.mscs.dal.ca/~selinger/md5collision/

Reply:
If you can get the same md5 for two different files, I think we have celebration instead of a problem on our hands.